frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -323,10 +320,15 @@ const Graph: React.FC<GraphProps> = React.memo(
 
     const xAxisTickCount = Math.round(width / 100);
     const xAxisTickShowSeconds = span < xAxisTickCount * ONE_MINUTE;
-    const xAxisTickFormat =
-      span > parsePrometheusDuration('1d')
-        ? (d) => `${formatDate(d)}\n${twentyFourHourTime(d)}`
-        : (d) => twentyFourHourTime(d, xAxisTickShowSeconds);
+    const xAxisTickFormat = (d) => {
+      if (span > parsePrometheusDuration('1d')) {
+        return dateTimeFormatter.format(d);
```

Comment:
I notice this causes the axis labels to overlap. Maybe we can keep a `${date}\n${time}` approach here?

![screenshot](https://user-images.githubusercontent.com/460802/112861298-a4b76d00-90ef-11eb-8af5-115c006870c9.png)

Reply:
Thanks, yeah. I somehow missed that there was a newline bulk replacing these. I'll roll that change back.